fn/operate/runner_pools.md

```diff
@@ -6,96 +6,183 @@ You can run a load-balanced setup for fn to route requests to a group of one or
 
 ## Starting the components (as regular processes)
 
+### Persisted storage
+
+For the particular example we'd use MySQL 5.7.22:
+
+```bash
+docker run --name fn-mysql \
+        -p 3306:3306 \
+        -e MYSQL_DATABASE=funcs \
+        -e MYSQL_ROOT_PASSWORD=root \
+        -d mysql:5.7.22
+```
+
+### DNS name requirement
+
+For the following deployment there's a need in the following DNS names:
+
+ - runner LB node DNS name: `lb.fn.local`
+ 
+Please note, this DNS name is an example!
+
+
 ### API server
 
 ```bash
-FN_NODE_TYPE=api ./fnserver
+docker run --name fn-api \
+        -d \
+        -e FN_NODE_TYPE=api \
+        -e FN_LOG_LEVEL=DEBUG \
+        -e FN_PUBLIC_LB_URL="http://lb.fn.local:8181" \
+        -e FN_DB_URL="mysql://root:root@tcp(fn-mysql:3306)/funcs" \
+        --link fn-mysql:fn-mysql \
+        -p 8080:8080 \
+        fnproject/fnserver:latest
 ```
 
 ### Runners
 
 ```bash
-mkdir /tmp/runnerdata
-# first runner
-FN_NODE_TYPE=pure-runner FN_PORT=8082 FN_GRPC_PORT=9190 ./fnserver
-# on another terminal, start a second runner
-FN_NODE_TYPE=pure-runner FN_PORT=8083 FN_GRPC_PORT=9191 ./fnserver
+docker run --name fn-runner-1 \
+        --privileged \
+        -d \
+        -e FN_NODE_TYPE="pure-runner" \
+        -e FN_LOG_LEVEL=DEBUG \
+        -e FN_IOFS_DOCKER_PATH=${HOME}/.fn/iofs \
+        -e FN_IOFS_PATH=/iofs \
+        -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
Don’t worry, it’s just an example, it’s not a recommendation rather than dummy sample. I’ll add oss helm chart doc soon here with some info regarding differences between running Fn on docker VS Fn on k8s.